lib/commons/rscommons/thiessen/shapes.py

```diff
@@ -108,14 +108,30 @@ def centerline_points(in_lines: Path, distance: float = 0.0, transform: Transfor
 
             if fields:
                 for field in fields:
-                    value = feat.GetField(field)
-                    props[field] = value
+                    divergence = feat.GetField('Divergence')
```

Comment:
This is probably fine but it does affect RVD so I just want to make sure we understand where it's being used.